i18n-helpers/src/bin/mdbook-xgettext.rs

```diff
@@ -56,6 +61,18 @@ fn create_catalog(ctx: &RenderContext) -> anyhow::Result<Catalog> {
     metadata.mime_version = String::from("1.0");
     metadata.content_type = String::from("text/plain; charset=UTF-8");
     metadata.content_transfer_encoding = String::from("8bit");
+    metadata
+}
+
+/// Transform a Book into a HashMap of (file_name, catalog) pairs.
+/// The depth will be used to crawl the chapters and sub-chapters
```

Comment:
Tiny thing: the docstrings are Markdown and start with a one-line description of the function. You can see the results of this with `cargo doc --open` locally:

```suggestion
/// Transform a Book into a HashMap of `(file_name, catalog)` pairs.
///
/// The depth will be used to crawl the chapters and sub-chapters
```

Reply:
Ooh~ I was not aware that this was how that was generated How do I rebuild this doc to reflect local changes? It seems to be for ToT and not based on my local branch, even when I `cargo build`